docs/getting-started/new-project-guide/go_lang.md

```diff
@@ -27,10 +27,48 @@ only. In that mode, fuzz targets for Go use the libFuzzer engine with native Go
 coverage instrumentation. Binaries compiled in this mode provide the same
 libFuzzer command line interface as non-Go fuzz targets.
 
+## Native Go 1.18 support
+
+OSS-fuzz supports fuzzers written for the native Go 1.18 engine. These fuzzers are built as libFuzzer binaries in a similar fashion as fuzzers written for the go-fuzz engine. Because of that, dictionaries and seed corpora should be handled in accordance with [the OSS-fuzz documentation](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#seed-corpus).
+Any number of arguments along with all types supported by the Go 1.18 engine are also supported by OSS-fuzz. Here is an example of a valid fuzzer with multiple arguments:
+
+```go
+package demofuzzing
+
+import (
+    "fmt"
+    "testing"
+)
+
+func FuzzDemo(f *testing.F) {
+    f.Fuzz(func(t *testing.T, data1 string, data2 uint32, data3 float64) {
+        fmt.Println(data1)
+        fmt.Println(data2)
+        fmt.Println(data3)
+    })
+}
+```
+
+Some requirements for native Go 1.18 fuzzers are:
+* The parameter to your fuzz function must be `f *testing.F`, ie, this is an example of an invalid function signature: `func FuzzDemo(fuzzer *testing.F`). 
```

Comment:
Could we be a bit smarter with the sed replacement to account for other variable names here?

Reply:
Friendly ping! @AdamKorcz

Reply:
The sed command has been updated here: https://github.com/google/oss-fuzz/pull/7055/commits/37bc302e08ab20f66fe840e7e6176b0749be1445

and the documentation here: https://github.com/google/oss-fuzz/pull/7055/commits/a2ae977f9a9d4f4ff4b30226312cce6a984ca37f

As a result, this fuzz function:

```go
func TestOurFuzzFunc(Ff0JIoij324ioOIjoi2j3io4iojIJO0fFF *testing.F) {
        return
}
```

would be replaced with:

```go
func TestOurFuzzFunc(Ff0JIoij324ioOIjoi2j3io4iojIJO0fFF *go118fuzzbuildutils.F) {
        return
}
```

thus removing the need for any specific name for the `*testing.F`